components/UpdatedDate/UpdatedDate.tsx

```diff
@@ -0,0 +1,14 @@
+import { ReactNode } from "react"
+import styles from "./UpdatedDate.module.css"
+
+interface IUpdatedDateSectionProps {
+	date: string
+}
+export const UpdatedDate = ({ date }: IUpdatedDateSectionProps) => (
```

Comment:
I think a more appropriate name would be PageUpdated

Reply:
Sure